schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/convert/ogcwxs-info-injection.xsl

```diff
@@ -554,7 +554,6 @@
                              select="."/>
       </xsl:for-each>
 
-      <xsl:apply-templates mode="copy" select="mri:resourceConstraints"/>
```

Comment:
One risk here is that we remove resourceConstraints if there is no wms:AccessConstraints (which is optional https://schemas.liquid-technologies.com/opengis/wms/1.3.0/?page=accessconstraints.html)

Reply:
Fixed in the commit below.